setup.py

```diff
@@ -1,19 +1,75 @@
 #!/usr/bin/env python
+"""Setup.py for eulxml package"""
 from distutils.command.build_py import build_py
+from distutils.command.clean import clean
+from distutils.command.sdist import sdist
+from distutils.core import Command
 import os
 import sys
-
 from setuptools import setup, find_packages
+import shutil
 
 import eulxml
 
-class build_py_with_ply(build_py):
-    '''Use ply to generate parsetab and lextab modules.'''
 
-    def run(self, *args, **kwargs):
+class GenerateXmlCatalog(Command):
```

Comment:
I wonder if these custom setup commands should be pulled into a submodule?  Seems a little weird to have so much logic in the setup.py